src/libraries/System.Management/Directory.Build.targets

```diff
@@ -0,0 +1,8 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.targets" />
+  <PropertyGroup>
+    <!-- This assembly is inbox in .NETFramework, ensure that the .NETStandard assembly
+         remains <= the desktop version -->
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'netstandard2.0'">4.0.0.0</AssemblyVersion>
```

Comment:
Ok nice. I can see that you're shipping a new version of System.Management in the backports.